src/test/java/com/baidu/hugegraph/api/VertexApiTest.java

```diff
@@ -209,6 +214,92 @@ public void testBatchCreate() {
         }
     }
 
+    @Test
+    public void testBatchUpdateStrategySum() {
+        BatchVertexRequest req = batchVertexRequest("price", 1, -1,
+                                                    UpdateStrategy.SUM);
+
+        this.graph().updateVertices(req).forEach(vertex -> {
+            Object price = vertex.properties().get("price");
+            Assert.assertTrue(price instanceof Number);
+            Assert.assertEquals(0, price);
+        });
+    }
+
+    @Test
+    public void testBatchUpdateStrategyBigger() {
+        BatchVertexRequest req = batchVertexRequest("price", 1, -1,
+                                                    UpdateStrategy.BIGGER);
+
+        this.graph().updateVertices(req).forEach(vertex -> {
+            Object price = vertex.properties().get("price");
+            Assert.assertTrue(price instanceof Number);
+            Assert.assertTrue((int) price > 0);
```

Comment:
Assert.assertEquals(1, price);

Reply:
The variable `i` is now appended to the test data, so the attribute of each element are not fixed values.
And due to the order of the collection is not fixed from server's response, cannot be one-to-one correspondence with index, like [set-property](https://github.com/hugegraph/hugegraph-client/blob/a2b555a4397754796e3ec46ab456977076caaf15/src/test/java/com/baidu/hugegraph/BaseClientTest.java#L267), [price-property](https://github.com/hugegraph/hugegraph-client/blob/a2b555a4397754796e3ec46ab456977076caaf15/src/test/java/com/baidu/hugegraph/BaseClientTest.java#L269), and others.
So shall we get rid of all the changes so that elements can have the same property values?